config/formats.js

```diff
@@ -721,8 +721,8 @@ let Formats = [
 		mod: 'gen7',
 		ruleset: ['[Gen 7] PU'],
 		banlist: [
-			'PU', 'Carracosta', 'Crabominable', 'Exeggutor-Base', 'Gorebyss', 'Jynx', 'Musharna',
-			'Raticate-Alola', 'Raticate-Alola-Totem', 'Throh', 'Turtonator', 'Ursaring', 'Victreebel', 'Zangoose',
+			'PU', 'Carracosta', 'Crabominable', 'Exeggutor-Base', 'Gorebyss', 'Jynx', 'Musharna', 'Raticate-Alola',
+			'Raticate-Alola-Totem', 'Throh', 'Turtonator', 'Type: Null', 'Ursaring', 'Victreebel', 'Zangoose',
```

Comment:
This is unrelated, but why is the format explicitly banning Raticate-Alola-Totem? This Pokemon should be illegal either way.

Reply:
Last I checked Totems weren't automatically banned with base formes.

Reply:
Totems are banned with base formes, but in the case of alolan totems, their base formes are the base forme of the alolan pokemon as well, so they dont get automatically banned.